pkg/cmd/roachtest/roachtestutil/mixedversion/mixedversion.go

```diff
@@ -1011,4 +1052,20 @@ func assertValidTest(test *Test, fatalFunc func(...interface{})) {
 		)
 		fatalFunc(errors.Wrap(err, "mixedversion.NewTest"))
 	}
+
+	currentVersion := clusterupgrade.CurrentVersion()
+	msv := test.options.minimumSupportedVersion
+	// The minimum supported version should be from an older major
+	// version or, if from the same major version, from an older minor
+	// version.
+	validVersion := msv.Major() < currentVersion.Major() ||
+		(msv.Major() == currentVersion.Major() && msv.Minor() < currentVersion.Minor())
```

Comment:
Shouldn't be `msv.Minor() <= currentVersion.Minor()`?

Reply:
That would allow the case where the minimum supported version and the current version differ only on patch/prerelese, i.e., they are part of the same release series. This means that this "upgrade" would not involve running migrations or changing the cluster version.

Since everything else in `mixedversion` is designed around upgrading from one series to the next, I felt like it's best to disallow this scenario at this moment.

Reply:
Makes sense; maybe the error message below could be made more precise in case the difference comes down to patch versions.

Reply:
Changed error to say `minimum supported version should be from an older release series than current version`.